src/components/modules/banners/digitalSubscriptions/DigitalSubscriptionsBanner.tsx

```diff
@@ -23,27 +23,66 @@ import {
 } from './digitalSubscriptionsBannerStyles';
 import { BannerProps } from '../../../../types/BannerTypes';
 import { setSubscriptionsBannerClosedTimestamp } from '../localStorage';
+import { addTrackingParams, createClickEventFromTracking } from '../../../../lib/tracking';
+
+const subscriptionUrl = 'https://support.theguardian.com/subscribe/digital';
```

Comment:
Do we need different stage versions of these urls?

Reply:
Yep I should've added a `// TODO` here, as I also think this could be an environment variable, based on whether you're on `PROD` or `CODE`. I checked our `frontend` banners and we use the same `PROD` value for the URL on both `PROD` or `CODE`, so it's not a step backwards at least.